integration-tests/src/test/java/com/datastax/oss/driver/mapper/ComputedIT.java

```diff
@@ -0,0 +1,306 @@
+  }
+
+  @Test
+  public void should_return_computed_values_in_select() {
+    ComputedDao computedDao = mapper.computedDao(sessionRule.keyspace());
+    int key = keyProvider.incrementAndGet();
+
+    long time = System.currentTimeMillis() - 1000;
+    ComputedEntity entity = new ComputedEntity(key, 1, 2);
+    computedDao.saveWithTime(entity, 3600, time);
+
+    ComputedEntity retrievedValue = computedDao.findById(key, 1);
+    assertThat(retrievedValue.getId()).isEqualTo(key);
+    assertThat(retrievedValue.getcId()).isEqualTo(1);
+    assertThat(retrievedValue.getV()).isEqualTo(2);
+    assertThat(retrievedValue.getTtl()).isEqualTo(3600);
+    assertThat(retrievedValue.getWritetime()).isEqualTo(time);
+  }
+
+  @Test
+  public void should_not_include_computed_values_in_delete() {
+    // should not be the case since delete operates on primary key..
+    ComputedDao computedDao = mapper.computedDao(sessionRule.keyspace());
+    int key = keyProvider.incrementAndGet();
+
+    ComputedEntity entity = new ComputedEntity(key, 1, 2);
+    computedDao.save(entity);
+
+    // retrieve values so computed values are present.
+    ComputedEntity retrievedValue = computedDao.findById(key, 1);
+
+    computedDao.delete(retrievedValue);
+
+    assertThat(computedDao.findById(key, 1)).isNull();
+  }
+
+  @Test
+  public void should_not_include_computed_values_in_SetEntity() {
+    ComputedDao computedDao = mapper.computedDao(sessionRule.keyspace());
+    int key = keyProvider.incrementAndGet();
+
+    CqlSession session = sessionRule.session();
+    PreparedStatement preparedStatement =
+        session.prepare("INSERT INTO computed_entity (id, c_id, v) VALUES (?, ?, ?)");
+    BoundStatementBuilder builder = preparedStatement.boundStatementBuilder();
+    ComputedEntity entity = new ComputedEntity(key, 1, 2);
+    BoundStatement statement = computedDao.set(builder, entity).build();
+    session.execute(statement);
+
+    // retrieve values to ensure was successful.
+    ComputedEntity retrievedValue = computedDao.findById(key, 1);
+
+    assertThat(retrievedValue.getId()).isEqualTo(key);
+    assertThat(retrievedValue.getcId()).isEqualTo(1);
+    assertThat(retrievedValue.getV()).isEqualTo(2);
+  }
+
+  @Test
+  public void should_return_computed_values_in_GetEntity() {
+    ComputedDao computedDao = mapper.computedDao(sessionRule.keyspace());
+    int key = keyProvider.incrementAndGet();
+
+    long time = System.currentTimeMillis() - 1000;
+    ComputedEntity entity = new ComputedEntity(key, 1, 2);
+    computedDao.saveWithTime(entity, 3600, time);
+
+    CqlSession session = sessionRule.session();
+    // query with the computed values included.
+    // since we the mapper expects the result name to match the property name, we used aliasing
+    // here.
+    ResultSet result =
+        session.execute(
+            SimpleStatement.newInstance(
+                "select id, c_id, v, writetime(v) as writetime, ttl(v) as myttl from "
```

Comment:
Looking at this definition:
``` @CqlName("myttl")
    @Computed("ttl(v)")
    private int ttl;
```
Don't you think it is a little bit confusing? 
If the select query is returning `mytll` alias maybe it will be enough to do: `@Compued(ttl(myttl))`?

Reply:
i'm intentionally doing this to test that the `@CqlName` annotation is used over the property name:

```java

    @CqlName("myttl")
    @Computed("ttl(v)")
    private int ttl;
```

I'll update the comment to reflect that.